src/Core/MakeModelMenu.elm

```diff
@@ -171,8 +183,19 @@ update msg model =
 
                         _ ->
                             ( RemoteData.Loading, Cmd.none )
+
+                make = findPreselectedMake model.preselectedMakeSlug sortedFilteredResponse
+                newModel = { model | state = MakeSelection sortedFilteredResponse }
             in
-                ( { model | state = MakeSelection sortedFilteredResponse }, Cmd.none )
+                case model.preselectedMakeSlug of
```

Comment:
This looks like a `Maybe.map2` I think. You want to do something when two maybes both have a value, and if either is `Nothing` then do something else. So
`Maybe.map2 (\preselected make -> (update (MakeSelected make) newModel)) model.preselectedMakeSlug make |> Maybe.withDefault (newModel, Cmd.none)`